Student/StudentUITests/Tests/StudentTest.swift

```diff
@@ -45,11 +41,8 @@ class StudentTest: XCTestCase {
             app!.launch()
         }
         host.reset()
-    }
-
-    func launch(_ route: String, as user: AuthUser) {
-        host.logIn(domain: seedClient.baseURL.host!, token: user.token)
-        show(route)
+        sleep(1) // FIXME: Remove this and fix flakiness better.
+        // This sleep helps ensure old views got cleaned up, so EG2 doesn't find them accidentally.
```

Comment:
Interesting... this is worrisome.

Reply:
I think most of the flaky things were webviews. It must take a second after they are popped off the nav before XCUIElement & EG2 stop finding them.